package.json

```diff
@@ -4,20 +4,32 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "build": "prisma migrate reset --force && prisma generate && next build",
```

Comment:
This resets the database, runs the migrations and loads the sample data on every new deployment currently. Did it like that for now to test that seeding the database as part of the Netlify build works but for the live version we might not want to reset the database every time.